### PR TITLE
RHDEVDOCS-2381 Document prunning of PipelineRun and TaskRun

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.8`, `enterprise-4.9`, `enterprise-4.10`
- **JIRA issues**: [Document prunning of PipelineRun and TaskRun](https://issues.redhat.com/browse/RHDEVDOCS-2381)
- **Preview pages**: See the second example in the following sections:
  - [pipelinerun delete](https://deploy-preview-35460--osdocs.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/op-tkn-reference?utm_source=github&utm_campaign=bot_dp#pipelinerun-delete)
  - [taskrun delete](https://deploy-preview-35460--osdocs.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/op-tkn-reference?utm_source=github&utm_campaign=bot_dp#taskrun-delete)
- **SME+QE review**: @VeereshAradhya 
- **Peer-review**: @rolfedh

### CODE REVIEW

File: modules/op-tkn-pipeline-management.adoc [hunk 1]
```diff
@@ -16,45 +16,45 @@ $ tkn pipeline --help
...
 ----
 $ tkn pipeline delete mypipeline -n myspace
 ----
 
 == pipeline describe
-Describe a Pipeline.
+Describe a pipeline.
 
-.Example: Describe `mypipeline` Pipeline
+.Example: Describe `mypipeline` pipeline
```
Comment (rolfedh) [inline modules/op-tkn-pipeline-management.adoc:30]:
```suggestion
.Example: Describe the `mypipeline` pipeline
```

File: modules/op-tkn-pipeline-management.adoc [hunk 2]
```diff
@@ -16,45 +16,45 @@ $ tkn pipeline --help
...
 [source,terminal]
 ----
 $ tkn pipeline describe mypipeline
 ----
 
 == pipeline list
-List Pipelines.
+List pipelines.
```
Comment (rolfedh) [inline modules/op-tkn-pipeline-management.adoc:37]: Maybe 
```suggestion
Display a list of pipelines.
```

File: modules/op-tkn-pipeline-management.adoc [hunk 3]
```diff
@@ -16,45 +16,45 @@ $ tkn pipeline --help
...
-.Example: Display a list of Pipelines
+.Example: Display a list of pipelines
 [source,terminal]
 -----
 $ tkn pipeline list
 -----
 
 == pipeline logs
-Display Pipeline logs for a specific Pipeline.
+Display pipeline logs for a specific pipeline.
```
Comment (rolfedh) [inline modules/op-tkn-pipeline-management.adoc:46]:
```suggestion
Display the logs for a specific pipeline.
```

File: modules/op-tkn-pipeline-management.adoc [hunk 4]
```diff
@@ -16,45 +16,45 @@ $ tkn pipeline --help
...
 
-.Example: Stream live logs for the `mypipeline` Pipeline
+.Example: Stream live logs for the `mypipeline` pipeline
```
Comment (rolfedh) [inline modules/op-tkn-pipeline-management.adoc:48]:
```suggestion
.Example: Stream the live logs for the `mypipeline` pipeline
```
This example changes the verb from "display" to "stream." If stream means "to display a continuously updating list of log messages," I would update line 46 to something like "Display a stream of log messages for a specific pipeline."

File: modules/op-tkn-pipeline-management.adoc [hunk 5]
```diff
@@ -16,45 +16,45 @@ $ tkn pipeline --help
...
 ----
 $ tkn pipeline logs -f mypipeline
 ----
 
 == pipeline start
-Start a Pipeline.
+Start a pipeline.
 
-.Example: Start `mypipeline` Pipeline
+.Example: Start `mypipeline` pipeline
```
Comment (rolfedh) [inline modules/op-tkn-pipeline-management.adoc:57]: Please review the IBM SG section on "Articles," which states "Articles increase clarity and ease translation. The articles “a” and “an” are indefinite articles. Use them to refer to things in the abstract or not in the particular. “An offering” or “a product” means any offering or any product, as opposed to a specific offering or product. “The” is a definite article. Use it to refer to a particular thing or things, meaning that “the offering” or “the product” is about that particular offering or product. Do not omit the articles a, an, and the."

Please review and update all of these revisions accordingly.

Comment (rolfedh) [reply modules/op-tkn-pipeline-management.adoc:57]:
```suggestion
.Example: Start the `mypipeline` pipeline
```

File: modules/op-tkn-task-run.adoc [hunk 6]
```diff
@@ -26,35 +26,42 @@ $ tkn taskrun cancel mytaskrun -n myspace
 == taskrun delete
 Delete a TaskRun.
 
-.Example: Delete `mytaskrun1` and `mytaskrun2` TaskRuns from a namespace
+.Example: Delete `mytaskrun1` and `mytaskrun2` task runs from a namespace
```
Comment (rolfedh) [inline modules/op-tkn-task-run.adoc:29]:
```suggestion
.Example: Delete the `mytaskrun1` and `mytaskrun2` task runs from a namespace
```

File: modules/op-tkn-task-run.adoc [hunk 7]
```diff
@@ -26,35 +26,42 @@ $ tkn taskrun cancel mytaskrun -n myspace
...
 ----
 $ tkn taskrun describe mytaskrun -n myspace
 ----
 
 == taskrun list
-List TaskRuns.
+List task runs.
 
-.Example: List all TaskRuns in a namespace
+.Example: List all task runs in a namespace
```
Comment (rolfedh) [inline modules/op-tkn-task-run.adoc:54]:
```suggestion
.Example: List all the task runs in a namespace
```

**Review by rolfedh — APPROVED** [review]
Pre-approved with the expectation that you'll apply the suggestions.

**Comment by sounix000** [comment]
@rolfedh - Thanks for the review. I have incorporated your suggestions.

**Pull request merged**